web/public/js/directives/cd-picture-grid/style.less

```diff
@@ -1,49 +1,99 @@
 .cd-picture-grid {
-    display: flex;
-    flex-wrap: wrap;
-    justify-content: center;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
 
-    &__item {
-      flex: 1;
-      min-width: 208px;
-      max-width: 208px;
-      padding: 16px 32px;
-      cursor: pointer;
+  &__item {
+    flex: 1;
+    min-width: 208px;
+    max-width: 208px;
+    padding: 16px 32px;
+    cursor: pointer;
 
-      &:hover {
-        text-decoration: none;
+    &:hover {
+      text-decoration: none;
 
-        .cd-picture-grid__picture {
-          box-shadow: 0 0 0 10px @cd-purple, 0 0 0 0 @cd-blue;
-        }
+      .cd-picture-grid__picture {
+        box-shadow: 0 0 0 10px @cd-purple, 0 0 0 0 @cd-blue;
       }
+    }
 
-      &--selected, &--selected:hover {
-        .cd-picture-grid__picture {
-          box-shadow: 0 0 0 0 @cd-purple, 0 0 0 10px @cd-blue;
-        }
+    &--selected, &--selected:hover {
+      .cd-picture-grid__picture {
+        box-shadow: 0 0 0 0 @cd-purple, 0 0 0 10px @cd-blue;
       }
     }
+  }
 
-    &__picture {
-      border-radius: 100%;
-      padding-top: 100%;
-      background-size: cover;
-      background-repeat: no-repeat;
-      background-position: center;
-      transition: box-shadow 0.2s ease-in-out;
-    }
+  &__picture {
+    border-radius: 100%;
+    padding-top: 100%;
+    background-size: cover;
+    background-repeat: no-repeat;
+    background-position: center;
+    transition: box-shadow 0.2s ease-in-out;
+  }
+
+  &__caption {
+    text-align: center;
+    margin-top: 16px;
+    font-size: 16px;
+  }
 
-    &__caption {
-      text-align: center;
-      margin-top: 16px;
-      font-size: 16px;
+  @media screen and (max-width: @screen-xs-max) {
+    &__item {
+      min-width: 144px;
+      max-width: 144px;
     }
+  }
 
-    @media screen and (max-width: @screen-xs-max) {
+  &--wide-card {
+    .cd-picture-grid {
       &__item {
-        min-width: 144px;
-        max-width: 144px;
+        display: flex;
+        padding: 16px;
+        min-width: 368px;
```

Comment:
Dunno if variables really make sense here... They're specific to this component, just picked based on how it looked.